presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -123,6 +172,28 @@ public TableScanNode tableScan(List<Symbol> symbols, Map<Symbol, ColumnHandle> a
         );
     }
 
+    public TableScanNode tableScan(String tableName, Map<String, String> symbolMap)
+    {
+        QualifiedObjectName name = new QualifiedObjectName(session.getCatalog().get(), session.getSchema().get(), tableName);
+        TableHandle tableHandle = metadata.getTableHandle(session, name).orElse(new TableHandle(
```

Comment:
The problem is that PlanPatternMatcher will check session and table metadata for the table/column. So we cannot use arbitrary table here.